scss/themes/default/_theme-colors.scss

```diff
@@ -3,487 +3,490 @@
+        "primary-focus": rgba($slate-400, 0.5),
+        "primary-inverse": $white,
+        "switch-thumb-box-shadow": 0 0 0 rgba(0, 0, 0, 0),
+      ),
+    ),
+    "violet": (
+      "dark": (
+        "text-selection-color": rgba($violet-350, 0.1875),
+        "primary": $violet-350,
+        "primary-background": $violet-600,
+        "primary-underline": rgba($violet-350, 0.5),
+        "primary-hover": $violet-250,
+        "primary-hover-background": $violet-550,
+        "primary-focus": rgba($violet-350, 0.375),
+        "primary-inverse": $white,
+        "switch-thumb-box-shadow": 0 0 0 rgba(0, 0, 0, 0),
+      ),
+      "light": (
+        "text-selection-color": rgba($violet-400, 0.25),
+        "primary": $violet-550,
+        "primary-background": $violet-600,
+        "primary-underline": rgba($violet-550, 0.5),
+        "primary-hover": $violet-650,
+        "primary-hover-background": $violet-650,
+        "primary-focus": rgba($violet-400, 0.5),
+        "primary-inverse": $white,
+        "switch-thumb-box-shadow": 0 0 0 rgba(0, 0, 0, 0),
+      ),
+    ),
+    "yellow": (
+      "dark": (
+        "text-selection-color": rgba($yellow-350, 0.1875),
+        "primary": $yellow-350,
+        "primary-background": $yellow-100,
+        "primary-underline": rgba($yellow-350, 0.5),
+        "primary-hover": $yellow-250,
+        "primary-hover-background": color.mix($yellow-100, $yellow-50),
+        "primary-focus": rgba($yellow-350, 0.375),
+        "primary-inverse": $black,
+        "switch-thumb-box-shadow": 0 0 0.5rem rgba($black, 0.25),
+      ),
+      "light": (
+        "text-selection-color": rgba($yellow-400, 0.25),
+        "primary": $yellow-550,
+        "primary-background": $yellow-100,
+        "primary-underline": rgba($yellow-550, 0.5),
+        "primary-hover": $yellow-650,
+        "primary-hover-background": $yellow-150,
+        "primary-focus": rgba($yellow-400, 0.5),
+        "primary-inverse": $black,
+        "switch-thumb-box-shadow": 0 0 0.5rem rgba($black, 0.25),
+      ),
+    ),
+    "zinc": (
+      "dark": (
+        "text-selection-color": rgba($zinc-350, 0.1875),
+        "primary": $zinc-350,
+        "primary-background": $zinc-550,
+        "primary-underline": rgba($zinc-350, 0.5),
+        "primary-hover": $zinc-250,
+        "primary-hover-background": $zinc-500,
+        "primary-focus": rgba($zinc-350, 0.375),
+        "primary-inverse": $white,
+        "switch-thumb-box-shadow": 0 0 0 rgba(0, 0, 0, 0),
+      ),
+      "light": (
+        "text-selection-color": rgba($zinc-400, 0.25),
+        "primary": $zinc-550,
+        "primary-background": $zinc-550,
+        "primary-underline": rgba($zinc-550, 0.5),
+        "primary-hover": $zinc-650,
+        "primary-hover-background": $zinc-600,
+        "primary-focus": rgba($zinc-400, 0.5),
+        "primary-inverse": $white,
+        "switch-thumb-box-shadow": 0 0 0 rgba(0, 0, 0, 0),
+      ),
     ),
   ),
+  $color-mappings

```

Comment:
```suggestion
  settings.$color-mappings
```

Reply:
Would we need to use the `settings.$theme-color` in that `get()` function as well?

Since `@use` started rolling out, I've always used the `@use 'file' as *;` way to get the variables. I searched around briefly to see the benefits of both, and I can see why we could use `file.$variable` but haven't seen anything noting whats the proper way to do that.

Reply:
Yeah I think so